### ISSUE TITLE
Dotted half-note plays back as quarter note

### ISSUE THREAD

**Issue by doctorscove** [musescore/MuseScore#17936]
### Issue type

General playback bug

### Bug description

I have attached an image of the score referenced below. Developers who want to download the .mscz file can go here: https://musescore.org/en/node/351032#comment-1193890

In measure 6 (the lead-in measure is considered measure 1), the dotted half "C" on the lower clef gets played back as a quarter note. This is not the case with other dotted half notes on the bottom clef. I experimented and discovered that after removing all or part of the upper voice (on the lower clef), that the dotted half "C" gets played for a full three beats. Something in voice 1 is causing the playback function to truncate the 3-beat dotted half "C" to a 1-beat quarter note. This score is from Frederick Noad's Solo Guitar Playing book.
<img width="929" alt="MuseScore" src="https://github.com/musescore/MuseScore/assets/66226057/11cfc534-a216-4453-b355-9c1ee478b754">

### Steps to reproduce

Please see the Bug description.

### MuseScore Version

4.0.2.230651546 / Revision: dbe7c6d

### Regression

I don't know

### Operating system

macOS Monterey 12.6.6

**Comment by DmitryArefiev** [comment]
@doctorscove I **can't reproduce** it on my side (4.0.2, Win10/Mac13)

Are you sure is dotted half C played as a quarter and not E (which is quarter)?

Here is my result:


https://github.com/musescore/MuseScore/assets/90187801/1fc0539f-6564-47ee-bf39-f30d398e803b

**Comment by MarcSabatella** [comment]
I think it's an auditory illusion.  The C does decay fairly quickly so that it's pretty soft by beat 2, but close listening with headphones tells me it's still there, and that much more obviously so if you reduce the velocity of the other notes in the measure.

**Comment by doctorscove** [comment]
Hello everyone, I listened carefully to @DmitryArefiev recording and am glad someone else besides me noticed that the 3-beat dotted half-note "C" is getting played back as a quarter note. I did my own experiment - in that same measure (6) I removed the eighth-note sequence C-B-A-G and played it back. That measure has no sound at all after the first beat (quarter note). So this confirms to me that something is wrong.
As for assigning this issue to me in GitHub, I'm not sure what I am supposed to do. I am certainly not a developer (at work I am a SW developer).
At this point I will assume MuseScore 4.0 is a work in progress with lots of bugs to be ironed out. To avoid offending anyone, I will point out that in my own SW development world, when we release major upgrades, oodles of problems show up despite our best efforts in design, coding and test.
If someone wants to close this issue they can. Others will find the problem (if it is confirmed to be an issue again).
